cpp/oneapi/dal/algo/pca/test/batch.cpp

```diff
@@ -54,6 +55,20 @@ class pca_batch_test : public te::algo_fixture {
         CAPTURE(component_count);
         const table x = data.get_table(this->get_policy(), data_table_id);
 
+        const auto meta = x.get_metadata();
+        if (meta.get_feature_type(0) == feature_type::nominal) {
+            fmt::print("it works1!");
+        }
+        if (meta.get_feature_type(2) == feature_type::nominal) {
+            fmt::print("it works2!");
+        }
+        if (meta.get_feature_type(3) == feature_type::nominal) {
+            fmt::print("it works3!");
+        }
+        if (meta.get_feature_type(4) == feature_type::ordinal) {
+            fmt::print("it works4!");
+        }
```

Comment:
This looks like a silent test on table metadata features.That's not the right place for such code - if you want to have tests on metadata, create separate tests.